index.html

```diff
@@ -528,8 +528,12 @@
                 <p>Talent consultants and recruiters/matchmakers focused on supporting Michigan technology companies with their talent needs.</p>
               </li>
               <li>
-                <a href="https://angel.co/michigan">AngleList - Ann Arbor Startups</a>
-                <p>A listing of Ann Arbor startups, including job openings.</p>
+                <a href="https://angel.co/michigan">AngelList - Michigan Startups</a>
```

Comment:
The link for AngelList had the text "Ann Arbor Startups", but linked to https://angel.co/michigan, which is all Michigan startups. I updated the text here to instead say 'Michigan Startups', but we could also link to https://angel.co/ann-arbor instead and keep the old text.